static/src/javascripts/projects/common/modules/experiments/embed.js

```diff
@@ -0,0 +1,49 @@
+define([
+    'common/utils/$'
+], function (
+    $
+) {
+    var ACTIVE_CLASS = 'active';
+    var AMOUNT_CLASS = 'js-amount';
+
+
+    function init() {
+        $.forEachElement(('[data-amount]'), function(el){
+            el.addEventListener('click', function () {
+                var element = event.currentTarget;
+                var amount = element.getAttribute('data-amount');
+                select(element);
+                setAmount(amount);
+            });
+        });
+    }
+
+    var getSpecifiedParent = function(element, parentClass) {
```

Comment:
(subjective: feel free to ignore)
Would it be better to name these `getSpecifiedAncestor`  and `ancestorClass`?

$.js has a `ancestor` function that may be helpful